tests/unit/util/helpers.js

```diff
@@ -0,0 +1,139 @@
+import { expect } from 'vitest';
+
+function cardsMatch(card1, card2) {
+  if(card1 && card2){
+    return card1.rank === card2.rank && card1.suit === card2.suit;
+  }
+  return false;
+}
+
+function gameStateRowMatch(fixture, res) {
+  // If length is not equal
+  if (fixture.length !== res.length){
+    return false;
+  }
+ 
+  fixture.sort();
+  res.sort();
+  // Comparing each element of array
+  for (let i = 0; i < fixture.length; i++){
+    if (fixture[i] != res[i]){
+      return false;
+    }
+  }
+  return true;
+}
+
+function gameStateMatch( res, fixture) {
```

Comment:
i think we can replace both this function and the one for GameStateRow's with a basic expect().toEqual from vitest:

`expect(gameState1).toEqual(gameState2)`